addons/rearm/functions/fnc_getMagazineName.sqf

```diff
@@ -28,21 +28,23 @@ if (isNil "_magName") then {
 
     if ((_displayName select [0,6]) == "[CSW] ") then { _displayName = _displayName select [6]; };
 
-    GVAR(magazineNameCache) setVariable [_className, _displayName];
+    GVAR(magazineNameCache) set [_className, _displayName];
     GVAR(originalMagazineNames) pushBack _displayName;
     TRACE_2("Adding to cache",_className,_displayName);
 
     // go through all existing cache entries and update if there now are duplicates
+    private _xMagName = "";
```

Comment:
why `private` outside the for loop?

Reply:
Avoids a variable being created on every iteration, which is performance consuming. However, the extra performance it costs is negligible, so I'll revert this type of change.

Reply:
I had it in my head that this type of micro-optimisation made much more of a difference, hence trying to update the code.